src/components/common/table/ReadableTextField.tsx

```diff
@@ -0,0 +1,53 @@
+import { TextField, TextFieldProps, Typography, styled } from '@mui/material';
+
+import { buildEditableTableTextInputCy } from '@/config/selectors';
+
+type Props = {
+  readonly: boolean;
+  size?: TextFieldProps['size'];
+  multiline?: TextFieldProps['multiline'];
+  value: string | unknown;
+  rowId: string;
+  fieldName: string;
+  onChange: (value: string) => void;
+};
+
+// Set the same padding as for the text input to avoid
+// movements between Typography and Textfield texts.
+const ReadyonlyTextField = styled(Typography)({
+  padding: '8px 14px',
```

Comment:
```suggestion
  padding: 'theme.spacing(1)px theme.spacing(2)px',
```
but with a correct syntax aha